code/modules/vehicle/ambulance.dm

```diff
@@ -102,6 +102,10 @@
 	anchored = FALSE
 	throw_pressure_limit = INFINITY //Throwing an ambulance trolley can kill the process scheduler.
 
+/obj/structure/bed/amb_trolley/examine(mob/user)
+	. = ..()
+	to_chat(user, "<span class='notice'>Drag \the [src]'s sprite over the ambulance to (de)attach it.</span>")
```

Comment:
no `\the`

Reply:
will it put it there on it's own or something weird? If so why does it do that?